src/main/fc/fc_tasks.c

```diff
@@ -366,13 +388,19 @@ cfTask_t cfTasks[TASK_COUNT] = {
         .staticPriority = TASK_PRIORITY_HIGH,
     },
 
-    [TASK_BATTERY] = {
-        .taskName = "BATTERY",
-        .taskFunc = taskUpdateBattery,
-        .desiredPeriod = TASK_PERIOD_HZ(50),        // 50 Hz
+    [TASK_BATTERY_ALERTS] = {
+        .taskName = "BATTERY_ALERTS",
+        .taskFunc = taskBatteryAlerts,
+        .desiredPeriod = TASK_PERIOD_HZ(5),        // 5 Hz
         .staticPriority = TASK_PRIORITY_MEDIUM,
     },
 
+    [TASK_BATTERY_SENSORS] = {
```

Comment:
I was trying to keep FC code out of battery.c, but actually now I think about it it's only voltage.c and current.c I need to keep FC code out of.  e.g.  sensor board doesn't need a battery, it just needs to expose voltage and current sensors.  the 'armed' stuff is OK in battery.c.  I'll give it a go.